xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/SelectingXmlNodesWithXpathTest.cs

```diff
@@ -0,0 +1,123 @@
+using System.Diagnostics;
+using System.Xml;
+using System.Xml.Linq;
+using Xunit;
+
+namespace SelectingXmlNodesWithXpathTests
+{
+    public class SelectingXmlNodesWithXpathTest
+    {
+        XmlDocument Document { get; set; }
+        Dictionary<string, string> ExpectedResults { get; set; }

```

Comment:
We should add `private` to these for consistency and clarity. They both can also be made `get` only since they are only set in the constructor.